Src/Sarif.PatternMatcher/JsonLogicalLocationProcessor.cs

```diff
@@ -27,6 +27,9 @@ public class JsonLogicalLocationProcessor
         ///     - For arrays, one might report the whole array, but this gets "closer" to the match.
         ///     - For objects, spanning properties will get "closer" to the match.
         /// </remarks>
+        /// <param name="results">A collection of 'Result' objects produced by an analysis tool where each object contains many details such as RuleId, Kind, Level, Locations, etc.</param>
+        /// <param name="fileContents">This is the litteral JSON file contents.</param>
```

Comment:
>litteral [](http://example.com/codeflow?start=51&length=8)

this is is not spelled correctly.

'This is' is not typical phrasing for this sort of thing, just use sentences like 'The literal JSON file contents.'